docs/GettingStarted.rst

```diff
@@ -1151,17 +1151,17 @@ the link.
 .. _Sharing-news:
```

Comment:
Shouldn't we rename also this ?

Reply:
I had the same idea but didn't know how to rename it. WDYT of "working-with-news"

Reply:
just "news" ?

Reply:
OK. I'll do it in the next commit.